libs/Meact/createDomElement.ts

```diff
@@ -0,0 +1,38 @@
+import { Element } from "../../type/meact";
+
+export const Fragment = "fragment";
+
+export const createDomElement = (meactNode: Element): HTMLElement => {
+  const dom = document.createElement(
+    meactNode.type === Fragment ? "div" : meactNode.type
+  );
+
+  Object.keys(meactNode.props).forEach((key) => {
+    if (key !== "children" && meactNode.props[key]) {
```

Comment:
p4: 아래처럼 하면 early return 해서 조금 간단하게 바꿀 수 있을 것 같네요
```suggestion
    if (key === "children" || !meactNode.props[key]) return; 
```

Reply:
반영했습니다!